atlas-data-storage/src/main/java/uk/ac/ebi/gxa/data/NetCDFProxy.java

```
@@ -378,17 +378,20 @@ public final class NetCDFProxy implements Closeable {
         ArrayChar uVal = (ArrayChar)uVALVar.read();
 
         final LinkedList<KeyValuePair> list = new LinkedList<KeyValuePair>();
-        for (Object text : (Object[])uVal.make1DStringArray().get1DJavaArray(String.class)) {
-            final String[] data = ((String)text).split(NCDF_PROP_VAL_SEP_REGEX);
+        for (Object text : (Object[]) uVal.make1DStringArray().get1DJavaArray(String.class)) {
+	        final String[] data = ((String) text).split(NCDF_PROP_VAL_SEP_REGEX, -1);
             if (data.length != 2) {
-                throw new RuntimeException("Invalid uVAL element: " + text);
+                throw new AtlasDataException("Invalid uVAL element: " + text);
+            }
+
+            if (!"".equals(data[1])) {
+                list.add(new KeyValuePair(data[0], data[1]));
             }
```

Am I right that we silently ignore all the `uVAL`s where `"".equals(data[1])`? Is it as intended?